haystack/json-schemas/generate_schema.py

```diff
@@ -0,0 +1,21 @@
+import os
```

Comment:
Nit: can we create a new package (like `schema`) to hold this code and eventually remove the folder `json-schemas` for clarity?

Reply:
That will imply changes in the pipeline code: right now it expects `haystack/json-schemas/haystack-pipeline-main.schema.json` to exist. I can go ahead and change it, but maybe not in this PR

Reply:
Arf I see. Yes let's not do it here.